packages/formation/scripts/babel-transformer.js

```diff
@@ -1,3 +1,3 @@
-module.exports = require('babel-jest').createTransformer({
+module.exports = require('babel-jest').default.createTransformer({
```

Comment:
This fixes the original issue with `createTransformer` not being found.

https://github.com/facebook/jest/issues/11444#issuecomment-847909076